templates/terraform/custom_expand/binaryauthorization_attestor_name.erb

```diff
@@ -0,0 +1,40 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2019 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
+  // This method returns a full self link from a partial self link.
```

Comment:
Is it possible to replace this whole method with a line similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/6ab63fa7b3ff69e5475028989341f55383418c90/templates/terraform/custom_expand/repository_name_from_short_name.go.erb#L16 ?

I don't think we need a whole self link here, just `projects/{{project}}/attestors/{{attestor}}` according to the docs